src/mon/OSDMonitor.cc

```diff
@@ -7485,6 +7485,10 @@ int OSDMonitor::prepare_command_pool_set(const cmdmap_t& cmdmap,
       ss << "specified invalid mode " << val;
       return -EINVAL;
     }
+    if (osdmap.require_osd_release < CEPH_RELEASE_NAUTILUS) {
```

Comment:
s/CEPH_RELEASE_NAUTILUS/ceph_release_t::nautilus/

Reply:
When did this happen and is there any way for people to know about it?
Maybe we should add compile warnings when changing the way stuff like this is supposed to work.

Reply:
it was changed in #27855 . i am not sure what do you mean? do you mean a message to ceph-devel before merging that change?

> Maybe we should add compile warnings when changing the way stuff like this is supposed to work.

by deprecating the `CEPH_RELEASE_NAUTILUS` macro? i am afraid C++ does not offer a portable way to deprecate macros. maybe we could use `_Pragma` in C99, but i am not sure..